auto/modules/java

```diff
@@ -297,7 +297,7 @@ NXT_JAR_NAME=jetty-http
 . auto/modules/java_get_jar
 
 NXT_JAR_NAME=classgraph
-NXT_JAR_VERSION=4.8.90
+NXT_JAR_VERSION=4.8.98
```

Comment:
According to https://mvnrepository.com/artifact/io.github.classgraph/classgraph , version 4.8.102 released in January. Is there any reason to use 4.8.98 ?

Reply:
Missed that one, please use the latest, i.e. 4.8.102.  Thanks.